src/components/toast/toast-transitions.ts

```diff
@@ -1,21 +1,26 @@
 import { Animation } from '../../animations/animation';
 import { Transition } from '../../transitions/transition';
-
+import { ToastOptions } from './toast-options';
 
 export class ToastSlideIn extends Transition {
   init() {
     // DOM READS
     let ele = this.enteringView.pageRef().nativeElement;
     const wrapperEle = <HTMLElement> ele.querySelector('.toast-wrapper');
     let wrapper = new Animation(wrapperEle);
+    let opts: ToastOptions = this.enteringView.data;
+    let padding = 10;
 
-    if (this.enteringView.data && this.enteringView.data.position === TOAST_POSITION_TOP) {
+    if (opts && opts.position === TOAST_POSITION_TOP) {
       // top
       // by default, it is -100% hidden (above the screen)
       // so move from that to 10px below top: 0px;
-      wrapper.fromTo('translateY', '-100%', `${10}px`);
+      if (opts.statusBarPadding) {
+        padding += 20;
```

Comment:
We shouldn't hardcode the status bar padding at 20, since it can be changed via Sass variable here: https://github.com/driftyco/ionic/blob/master/src/platform/cordova.scss#L7

and with this issue added support per mode: https://github.com/driftyco/ionic/issues/8712